cms/djangoapps/contentstore/management/commands/backfill_course_outlines.py

```diff
@@ -40,24 +40,30 @@ def add_arguments(self, parser):
             action='store_true',
             help="Show course outlines that will be backfilled, but do not make any changes."
         )
+        parser.add_argument(
+            '--force',
+            action='store_true',
+            help="Force Outline re-generation for all Courses, not just missing ones."
```

Comment:
qq: will this take a long amount of time? if so, is that worth putting in the help text?

Reply:
The command itself just loads one celery task per supported course onto the queue as quickly as possible–that part takes seconds. Running all those celery tasks took about 20 and 40 mins last time, between our two big environments. I'm not opposed to putting a message in, but I don't think it's necessary...? In terms of backfill commands, this is one of the milder ones.

Reply:
Nah that seems like a relatively reasonable amount of time